public/coda/coding_progress.html

```diff
@@ -27,28 +28,34 @@
         };
         firebase.initializeApp(config);
     </script>
+
 </head>
 <body>
-    
+<nav class='nav'>
+    <div class="nav-wrapper">
+        <a href="https://www.africasvoices.org/wp-content/themes/africa/static/img/africas-voices-logo-pad.svg" class="brand-logo center">Operations Dashboard</a>
+            <ul class="right hide-on-med-and-down">
+            <li><a href="https://avf-dashboards.firebaseapp.com/coda/coding_progress.html">Message Coding</a></li>
+            </ul>
+    </div>
+</nav>    
 <script src="coding_progress.js"></script>
-
-<table>
+<div class="container">
+<table class='datatable'>
```

Comment:
Please could you indent this as it's enclosed within the div

Reply:
Noted